easylist_adult/adult_specific_hide.txt

```diff
@@ -861,3 +861,4 @@ redtube.com,tube8.com,tube8.es,tube8.fr,xvideos.com,youjizz.com,youporn.com,youp
 redtube.com,tube8.com,tube8.es,tube8.fr,xvideos.com,youjizz.com,youporn.com,youporngay.com#?#:-abp-properties(data:)
 redtube.com,tube8.com,tube8.es,tube8.fr,xvideos.com,youjizz.com,youporn.com,youporngay.com#?#:-abp-properties(image/)
 youporn.com#?#aside:-abp-has(a.ad-remove)
+sexhd.pics##.tx
```

Comment:
Wrong place, move the rule just above `boysfood.com##.txt-a-onpage`. They're sorted by ASCII code of the rule part (not domain).